.github/workflows/scheduled.yml

```diff
@@ -233,27 +314,71 @@ jobs:
           path: |
             /tmp/fuzzer_repro
 
-  linux-spark-aggregate-fuzzer-run:
-    name: "Spark Aggregate Fuzzer"
+  presto-bias-fuzzer:
+    name: Presto Bias Fuzzer
     runs-on: ubuntu-latest
+    container: ghcr.io/facebookincubator/velox-dev:centos8
     needs: compile
-    timeout-minutes: 60
+    if: ${{ needs.compile.outputs.presto_bias  == 'true' }}
+    timeout-minutes: 120
     steps:
 
-      - name: "Checkout Repo"
-        uses: actions/checkout@v4
+      - name: Download presto expression fuzzer
+        uses: actions/download-artifact@v4
         with:
-          ref: "${{ inputs.ref }}"
+          name: presto
+
+      - name: Download Signatures
+        uses: actions/download-artifact@v4
+        with:
+          name: signatures
+          path: /tmp/signatures
+
+      - name: Run Presto Expression Fuzzer
+        run: |
+          ls /tmp/signatures
+          mkdir -p /tmp/presto_fuzzer_repro/
+            chmod -R 777 /tmp/presto_fuzzer_repro
+            chmod +x velox_expression_fuzzer_test
+            ./velox_expression_fuzzer_test \
+                --seed ${RANDOM} \
+                --lazy_vector_generation_ratio 0.2 \
+                --assign_function_tickets  $(cat /tmp/signatures/presto_bias_functions) \
```

Comment:
what if presto_bias_functions is empty - shouldnt there be a check above ?

Reply:
An empty file will not be created due to [this](https://github.com/facebookincubator/velox/pull/9191/files#diff-39a3b59376c532c693c0d42e43e02109591e5772115b3648973cbe6db33b9f06L130). And thus this step won't run. Previously an empty file might have been created due to the way argsparse.FileType works (it opens the file when the args are parsed) but I removed that.